app/src/main/java/com/andela/helpmebuy/activities/PurchaseHistoryActivity.java

```diff
@@ -46,12 +46,13 @@ public void intializeView() {
     }
 
     public void fetchPurchaseRequest() {
-        new FirebaseCollection<>(Constants.PURCHASE_REQUEST + "/" + userId, PurchaseRequest.class).getAll(new DataCallback<List<PurchaseRequest>>() {
+        new FirebaseCollection<>(Constants.PURCHASE_REQUEST + "/" + userId, PurchaseRequest.class)
+                .getAll(new DataCallback<List<PurchaseRequest>>() {
```

Comment:
It is clearer, easier to read and also considered a good pratice having lines of code that are not too long